source/documentation/deploying_services/postgresql.md

```diff
@@ -271,6 +271,93 @@ cf update-service SERVICE_NAME -c '{"reboot": true, "force_failover": true}'
+
+You can add or remove extensions for a PostgreSQL service instance by:
+
+- creating a new service instance
+- updating an existing service instance
+- restoring a PostgreSQL service instance snapshot
+
+#### Create a new service instance
+
+You [create a new PostgreSQL service instance](#set-up-a-postgresql-service) by running `cf create-service`. You can enable extensions in this new service instance by running:
+
+```
+cf create-service SERVICE_NAME -c '{"enabled_extensions": ["EXTENSION_1","EXTENSION_2",...,"EXTENSION_N"]}'
+```
+
+where:
+
+- `SERVICE_NAME` is a unique descriptive name for this service instance
+- `EXTENSION_1...N` are the names of the extensions you want to enable
+
+#### Update an existing service instance
+
+You add or remove extensions on an existing PostgreSQL service instance by updating your service with the new list of extensions:
+
+```
+cf update-service SERVICE_NAME -c '{"enabled_extensions": ["EXTENSION_1","EXTENSION_2",...,"EXTENSION_N"]}'
+```
+
+For example, your PostgreSQL service instance is named `my-pg-service` and has 3 extensions enabled, `uuid-ossp`, `postgis` and `citext`. Run the following to remove `citext`:
+
+```
+cf update-service my-pg-service -c '{"enabled_extensions": ["uuid-ossp","postgis"]}'
+```
+
+If you add the `pg_stat_statements` extension to your service instance, this extension will not start until the GOV.UK PaaS team reboots your service instance.
+
+This reboot will cause some service downtime. You should schedule this downtime to minimise service disruption.
+
+#### Restore a service instance
+
+If you have a [paid PostgreSQL plan](#paid-plans-postgresql), the Amazon RDS system [backs up your PostgreSQL service data](#postgresql-service-backup) every night. This includes all extensions that were enabled at time of backup.
+
+When you [restore a PostgreSQL service instance from a snapshot](#restoring-a-postgresql-service-snapshot), that restored service instance will by default include these extensions.
+
+If the snapshot contains extensions that GOV.UK PaaS no longer supports, the restored service instance will still include these extensions. However, you cannot create new service instances with unsupported extensions.
+
+#### Add or remove extensions when restoring a service instance
+
+When you restore a service instance, you can change the enabled extensions in that restored service instance. To do this, create your service with a new list of extensions:
+
+```
+cf create-service postgres PLAN NEW_SERVICE_NAME -c '{"restore_from_latest_snapshot_of": "GUID", "enabled_extensions": ["EXTENSION_1","EXTENSION_2",...,"EXTENSION_N"]}'
+```
+where:
+
+- `PLAN` is the plan used in the original service instance
+- `NEW_SERVICE_NAME` is a unique, descriptive name for the restored service instance
+- `GUID` is the global unique identifier of the backed up service instance
+- `EXTENSION_1...N` are the names of the extensions you want to enable
```

Comment:
Reading this and the last section, I can see how it might be unclear that these are _additional_ extensions, and not an exclusive list as it usually would be.